frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1365,7 +1401,8 @@ function ReaderHighlight:onHoldRelease()
     local long_final_hold = false
     if self.hold_last_tv then
         local hold_duration = TimeVal:now() - self.hold_last_tv
-        if hold_duration > TimeVal:new{ sec = 3, usec = 0 } then
+        local long_hold_threshold = G_reader_settings:readSetting("highlight_long_hold_threshold", 3)
```

Comment:
You forgot to add the widget to configure this ? Or do you want this just manually configured (we have a few settings like that, depends on how essential they are).
You could add it in plugins/gestures.koplugin/main.lua (we have a `Long-press interval` `ges_hold_interval`), or in the "Long-press on text' Gesture menu, your choice.

Reply:
widget added.